LED Interface task 2

cag343 opened the pull request:
Updated version of the led_interface.py using the ChessPosition class and it's variables for the position

Commit:
Add files via upload

Updated version of the led_interface.py using the ChessPosition class and it's variables for the position

catvec commented on led_interface.py:
```diff
@@ -1,21 +1,22 @@
 
     def __init__(self):
-        pass
+        self.chess_pos = ChessPosition(self, col, row)
```
Making the class in your `__init__()` method should only be used for classes which represent systems you will need to control. Such as the LCD screen, or LEDs. 

The `ChessPosition` and `ChessMove` classes we made represent a position on the Chess board. So you  should use them whenever you need to represent a move. So for your class that would be in the `turn_on_led`, `turn_off_led`, `start_blinking_led`, and `stop_blinking_led` methods.

Delete this line.

catvec commented on led_interface.py:
```diff
@@ -1,21 +1,22 @@
-        pass
+        self.chess_pos = ChessPosition(self, col, row)
 
-    def turn_on_led(self, position):
```
Keep this line, all the methods should take a single `position` argument. This `position` argument will be a `ChessPosition` on the board.

catvec reviewed (changes requested):
See comments about usage of `ChessPosition` class.

Commit:
Updated led_interface.py

Updated it so that no classes are in the __init__ function and updated arguments to have a position argument instead of col and row

catvec reviewed (approved):
Excellent
